tests/ensemble_evaluator/test_data_marshaller_and_unmarsaller.py

```diff
@@ -0,0 +1,25 @@
+import datetime
+
+from cloudevents.http import from_json, to_json
+from cloudevents.http.event import CloudEvent
+
+from ert_shared.ensemble_evaluator.entity import serialization
+
+
+def test_data_marshaller_and_unmarshaller(ee_id=0, event_index=0):
```

Comment:
Not sure what those parameters mean? I am unfamiliar with that approach to writing tests.

Reply:
my bad, we don't need this parameters at all